Tools/Scripts/libraries/webkitapipy/webkitapipy/reporter.py

```diff
@@ -25,6 +25,13 @@ def emit_diagnostic(self, diag: Diagnostic) -> bool:
             return True
         return False
 
+    def demangle_name(self, diag: Diagnostic) -> str:
+        if diag.kind == SYMBOL:
+            # FIXME: Consider using c++filt and swift-demangle in addition to
+            # C-style namespacing.
+            return diag.name.removeprefix('_')
```

Comment:
`"Demangle" them to the raw form` in `allow.py` refers to adding the _ prefix to get the raw symbol name. Here, demangling seems to refer to the opposite (removing the _ prefix to get to the C-style name). Which definition of demangle is correct?